geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/collections/SizableOrderStatisticsTree.java

```diff
@@ -46,11 +49,15 @@
  * @author Rodion "rodde" Efremov
  * @version 1.6 (Feb 11, 2016)
  */
-public class OrderStatisticsTree<E extends Comparable<? super E>>
+public class SizableOrderStatisticsTree<E extends Comparable<? super E>>
```

Comment:
It probably doesn't need to be renamed, since we don't have a non-sizeable implementation of OrderedStatisticsSet, so it doesn't make sense to distinguish it like this. I'll put the name back how it was.